src/intervals/arithmetic.jl

```diff
@@ -155,6 +155,25 @@ end
 
 //(a::Interval, b::Interval) = a / b    # to deal with rationals
 
+if VERSION >= v"0.6.0-dev"
+    const filter = Iterators.filter
+end
+
+if VERSION < v"0.5"
+    min(x) = x
+    max(x) = x
```

Comment:
These are indeed backporting something that is in 0.5 onwards. It is not available in Compat, but probably should be. https://github.com/JuliaLang/Compat.jl/issues/313